internal/resolution/manifest/maven.go

```diff
@@ -301,6 +309,28 @@ func (m MavenManifestIO) mergeParents(ctx context.Context, result *maven.Project
 	return result.Interpolate()
 }
 
+// Maven looks for the parent POM first in 'relativePath',
+// then the local repository '../pom.xml',
+// and lastly in the remote repo.
+func MavenParentPOMPath(currentPath, relativePath string) string {
+	if relativePath == "" {
```

Comment:
there is some caveat here: we are not able to tell whether the relative path is not set (should default to `../pom.xml`) or set to an empty string (parent should be fetched from upstream).
but we still check if the parent pom.xml exists and its key matches what we want, so I think this might be fine...

Reply:
Maybe there's a change that can be made in the deps.dev repo to set a default value of `relativePath` to `../pom.xml` if it's unspecified?

Reply:
this can be investigated - we need to make sure we are able to differentiate the two situations.